f5_os_test/infrastructure.py

```diff
@@ -121,3 +131,48 @@ def setup_with_pool_member(setup_with_pool):
                      'protocol_port': 80}}
     member = nclientmanager.create_lbaas_member(pool_id, member_config)
     return nclientmanager, activepool, member
+
+
+@pytest.fixture
+def get_auth_config(request, keystoneclientmanager):
+    token = keystoneclientmanager.auth_ref['token']['id']
```

Comment:
Interesting. It's an auth token. I wouldn't refer to it as an 'id' because it is not referred to as such by the command line client. It could confuse someone who has used that client.